integration_test/gce/gce_testing.go

```diff
@@ -989,9 +989,8 @@ func prepareSLES(ctx context.Context, logger *log.Logger, vm *VM) error {
 		// --gpg-auto-import-keys is included to fix a rare flake where (due to
 		// a policy being installed already) there is a new key that needs to
 		// be imported.
-		// timezone-java was selected arbitrarily as a package that:
-		// a) can be installed from the default repos, and
-		// b) isn't installed already.
+		// To fix this, we force install a package. coreutils is arbitrarily
+		// chosen as it's all but guaranteed to be present.
 		_, zypperErr := RunRemotely(ctx, logger, vm, "sudo zypper --non-interactive --gpg-auto-import-keys refresh && sudo zypper --non-interactive install timezone-java")
```

Comment:
I suspect you meant to actually change this to `coreutils`, i.e.:
```suggestion
		_, zypperErr := RunRemotely(ctx, logger, vm, "sudo zypper --non-interactive --gpg-auto-import-keys refresh && sudo zypper --non-interactive install --force coreutils")
```

Reply:
Yep, I did mean to change it lol. copy paste mishap

Done in [1f79427](https://github.com/GoogleCloudPlatform/ops-agent/pull/1743/commits/1f79427556d6133cba3ec79ff915e404414bd62b)